inspire/base/templates/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -0,0 +1,325 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% from "format/record/Inspire_Default_HTML_general_macros.tpl" import record_cite_modal, record_abstract with context %}
+
+{% macro record_collection_heading() %}
```

Comment:
This should be part of the breadcrums.

They are automatically generated by the application all over Invenio so we should figure out how to best use them so that all pages are consistent on the place they are shown and how they are displayed.

The first part should actually be a link that goes back to the search results.

For the moment we could leave it like that but we need a different task to convert them into Breadcrums.